src/insights.ts

```diff
@@ -0,0 +1,59 @@
+import * as https from "https";
+import * as os from "os";
+
+import * as appInsights from "applicationinsights";
+
+export class TelemetryClient implements appInsights.TelemetryClient {
+	public config: any = {};
+
+	public channel = {
+		setUseDiskRetryCaching: (): void => undefined,
+	};
+
+	public trackEvent(options: appInsights.EventTelemetry): void {
```

Comment:
It'd be helpful to add a track source env var.

Then for images deployed on clouds like DigitalOcean we can set something like `TELEMETRY_SOURCE=digitalocean` then analyze it